packages/vscodium.rb

```diff
@@ -46,14 +46,14 @@ class Vscodium < Package
   depends_on 'sommelier' # L
 
   no_fhs
-  no_shrink
```

Comment:
This was added to speed up the installs on arm.  It's fine to remove but just keep this in mind.

Reply:
Added back.